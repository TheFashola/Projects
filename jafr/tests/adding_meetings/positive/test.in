2
Meet Batman atop precinct
22/03/23
22:22
n