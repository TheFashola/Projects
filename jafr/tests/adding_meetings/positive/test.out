Just a friendly reminder! You have these tasks to finish today.

These tasks need to be finished in the next three days!
- Budget Approval for Next Quarter by 04/09/23
- Pick Up Birthday Gift for Barbara by 04/09/23
- Draft Speech for Town Hall by 05/09/23
- Review Neighborhood Watch Applications by 05/09/23

You have the following meetings today!
- Task Force Briefing on Two-Face at 10:00

You have the following meetings scheduled over the next week!
- Town Hall Meeting on Public Safety on 04/09/23 at 18:00
- Meeting with Mayor on 04/09/23 at 13:00

What would you like to do?
1. Complete tasks
2. Add a new meeting.
3. Share a task.
4. Share a meeting.
5. Change Jafr's master directory.
6. Exit
2
Please enter a meeting description:Meet Batman atop precinct

Please enter a date:22/03/23

Please enter a time:22:22

Ok, I have added Meet Batman atop precinct on 22/03/23 at 22:22.
Would you like to share this meeting? [y/n]: n
What would you like to do?
1. Complete tasks
2. Add a new meeting.
3. Share a task.
4. Share a meeting.
5. Change Jafr's master directory.
6. Exit